linux/autobuild/codeQL/auto_2_build_qt_project.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# autobuild_2_build: actual build process
+
+
+####################
+###  PARAMETERS  ###
+####################
+
+if [ ! -z "${1}" ]; then
```

Comment:
Not only is there code repeated for the same platform, it's repeated across platforms.

It does argue for restructuring the autobuild to be the top level with platform-specific code in subdirectories.  That would encourage the common code to be shared and only drop in the specifics where needed.

Reply:
I'm still in favor of that.

Reply:
sneak preview:
https://github.com/nefarius2001/jamulus/tree/autobuild_head
https://github.com/nefarius2001/jamulus/commit/784997c747c50b14dd5288b0219e9f57d2e3fc27

will merge to PR after successfull tests